conf/deployment/vsphere/upi_1az_rhcos_vmfs_3m_3w_3i.yaml

```diff
@@ -0,0 +1,19 @@
+---
+# This config is suppose to work on most of DCs we have.
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+  infra_nodes: True
+  ocs_operator_nodes_to_taint: 3
+ENV_DATA:
+  platform: 'vsphere'
+  deployment_type: 'upi'
+  worker_replicas: 6
+  master_replicas: 3
+  worker_num_cpus: '16'
+  master_num_cpus: '4'
+  master_memory: '16384'
+  compute_memory: '65536'
+  fio_storageutilization_min_mbps: 10.0
+REPORTING:
+  polarion:
+    deployment_id: 'xxx'
```

Comment:
Is there an actual polarion-id? If not let's remove this section.

These also need empty lines at the end of the file.

Reply:
Or have it commented out with TODO to add those polarion IDs and please open issue to not forger about it

Reply:
Done, added 2 polarion tests and updated with test ids